query/src/main/scala/filodb/query/exec/SelectRawPartitionsExec.scala

```diff
@@ -58,10 +80,14 @@ final case class SelectRawPartitionsExec(id: String,
                           queryConfig: QueryConfig)
                          (implicit sched: Scheduler,
                           timeout: FiniteDuration): Observable[RangeVector] = {
-    require(colIds.indexOfSlice(dataset.rowKeyIDs) == 0)
+    require(dataset.rowKeyIDs.forall(rk => !colIds.contains(rk)),
```

Comment:
Again prefer not to add this check here.  rowKeyIDs will be gone; and likely how to select columns will have to be redone in the near future anyways.

Reply:
Agree that this will change in the future. Simplification of column selection forced me to put this check here since it was quite ambiguous whether row keys should/shod-not be selected. I just fixed it and set the rules. I am open to changing this in a later PR.